app/javascript/components/UI/ShoutoutButton.js

```diff
@@ -1,12 +1,42 @@
-import React from 'react';
-import ShoutoutIcon from '../../../assets/./sys_svg/shoutout.svg'
+import React, {useEffect, useState} from 'react';
+import ShoutoutIcon from '../../../assets/./sys_svg/shoutoutNew.svg'
```

Comment:
@VitaliiShevchenko could you explain why we have two folders for images: `assets/images` and `assets/./sys_svg`?

Reply:
This is a directory with changed svg files. I specially store them separately so that no one sometimes deletes them or overwrites them. In next task to relative with shoutout button I 'll plan to unite/move them to the  /images